CHANGELOG.md

```diff
@@ -18,6 +18,7 @@
 ### Performance enhancements
 
 ### Bug fixes
+* The `include_self` parameter  was not being honored in `skbio.TreeNode.tips`. The scope of this bug was that if tips was called on a tip, it would always result in an empty list when unrolled.
```

Comment:
there are two spaces between "parameter" and "was"

"The scope of this bug was that if tips was called on a tip" -> "The scope of this bug was that if `TreeNode.tips` was called on a tip"

backticks needed on "list" too